backend/de.metas.business.rest-api-impl/src/main/java/de/metas/rest_api/invoicecandidates/impl/InvoicesRestControllerImpl.java

```diff
@@ -157,6 +159,28 @@ public InvoicesRestControllerImpl(
 		return ResponseEntity.notFound().build();
 	}
 
+	@ApiResponses(value = {
+			@ApiResponse(code = 200, message = "Invoice info retrieved"),
+			@ApiResponse(code = 401, message = "You are not authorized"),
+			@ApiResponse(code = 404, message = "No Invoice found")
```

Comment:
are these HTTP codes still valid for this method?

Reply:
yes